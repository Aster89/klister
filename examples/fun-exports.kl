#lang kernel
(import (shift kernel 1))
(import (shift kernel 2))
(import (shift kernel 3))




(meta (define a 2))

(meta (meta (define a 3)))

(meta (meta (meta (define a 4))))

(define a 1)

(define b a)
(meta (define b a))
(meta (meta (define b a)))
(meta (meta (meta (define b a))))

(example b)
(meta (example b))
(meta (meta (example b)))
(meta (meta (meta (example b))))

(export a)
(export (rename ([a b]) (shift 1 a)))
(export (shift 2 (prefix "the-" a)))
