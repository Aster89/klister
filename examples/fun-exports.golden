Signal 1
Signal 2
Signal 3
Signal 4
